<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSMARTCONTROL</class>
 <widget class="QDialog" name="CSMARTCONTROL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能家居控制</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="Qpush3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>76</width>
     <height>83</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>83</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>76</width>
     <height>83</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Qpush2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>76</width>
     <height>83</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>83</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>76</width>
     <height>83</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Qpush1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>76</width>
     <height>83</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>83</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>76</width>
     <height>83</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Qpush4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>76</width>
     <height>83</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>76</width>
     <height>83</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>76</width>
     <height>83</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>431</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/smart_home_curtain_open_1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>331</width>
     <height>135</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic/smart_home_air_condition_pic.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Qpush6">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>105</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Qpush5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
